docs/architecture-decision-records/0019-remove-encryption-from-notification-topic.md

```diff
@@ -0,0 +1,42 @@
+# 19. Remove encryption from notifications SNS topic

```

Comment:
1. Should there be an apostrophe after "n" in "notifications" in "on the notifications"?
2. Also, here and throughout this page, we use "SNS topic"; should it be "SNS topics"?

Reply:
1. Capitalised to make a name.
2. This ADR is about the single dr2-notifications SNS topic.